pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/elastic/search/ElasticSearchParticipantDto.java

```diff
@@ -34,11 +34,11 @@ public class ElasticSearchParticipantDto {
     private List<Map<String, Object>> workflows;
     private String status;
     private Dsm dsm;
+    private String queriedParticipantId; // optional, used for troubleshooting to report
```

Comment:
Nit: perhaps put this comment above `queriedParticipantId`. As is it appears to be two comments, the second applying to the getter.